.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<Do not remove this template and always check the preview before making the PR>
+
+### Related Issue #<number>
+
+Description: <provide a description> The title should be short and relevant.
+
+#### Checklist
+
+<just put a x between that square brackets to check that checkbox without spaces>
```

Comment:
Remove this line for enabling the functionality of the checklist.

Reply:
Done, thanks for suggesting this 🙌